MaterialLib/MPL/Properties/RelPermLiakopoulos.h

```diff
@@ -0,0 +1,77 @@
+/**
+ * \file
+ * \author Norbert Grunwald
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "BaseLib/ConfigTree.h"
+#include "MaterialLib/MPL/Property.h"
+
+namespace MaterialPropertyLib
+{
+class Medium;
+class Phase;
+class Component;
+/**
+ * \class RelPermLiakopoulos
+ * \brief Relative permeability function for Liakopoulos Benchmark
+ * \details This property must be a medium property, it
```

Comment:
It would be better to add its formula in this documentation.

Reply:
Agreed. I added the equations.